docs/pages/guides/linking.mdx

```diff
@@ -47,6 +47,36 @@ There are some URL schemes for core functionality that exist on every platform.
 | `tel`            | Open phone app, eg: `tel:+123456789`          |
 | `sms`            | Open SMS app, eg: `sms:+123456789`            |
 
+On newer Android versions, include the appropriate queries in the **AndroidManifest.xml** to open links. This can be done by [creating a config plugin](/config-plugins/plugins-and-mods/#create-a-plugin). For example:
```

Comment:
```suggestion
On newer Android versions, include the appropriate queries in the **AndroidManifest.xml** to open links. This can be done by [creating a config plugin](/config-plugins/plugins-and-mods/#create-a-plugin). For example, the config plugin below will enable linking to phone and email apps:
```